carma_v2x_msgs/msg/MobilityOperation.msg

```diff
@@ -4,7 +4,7 @@
 # CAVs at any desired frequency in order to accomplish a collaborative plan
 
 # standard header for all mobility messages
-carma_v2x_msgs/MobilityHeader  header
+carma_v2x_msgs/MobilityHeader  m_header
```

Comment:
I see "header" was changed to "m_header" for both ros1 and ros2 messages. I don't really have an issue with that since "header" is a sort of standard term in ros for std_msgs/Header meaning this new name is clearer, but I want to confirm this is intentional?

Reply:
yes rosbridge had problem with name header